src/Util/Frame.php

```diff
@@ -49,7 +60,7 @@ public function getFile()
     }
 
     /**
-     * @return int|null
+     * @return integer|null
```

Comment:
@sagikazarmark All other league repos use "int" instead of "integer". That's what I was meaning.

> Actually no, I don't. How is "standardclass" related to MyClass[]?

That was related to aliasing. "standardclass" might be how we read it, but it's wrong to document it like that in the same way that documenting "int" as "integer" is wrong.